chandra_aca/tests/test_maude_decom.py

```diff
@@ -63,7 +63,8 @@ def test_vcdu_0_raw():
 
     ref_data = test_data['686111007-686111017']['raw']
     data = maude_decom.get_raw_aca_packets(686111007, 686111017)
-    for key in ['TIME', 'MJF', 'MNF']:
+    assert np.all(np.isclose(data['TIME'], ref_data['TIME']))
```

Comment:
@jeanconn's point was a good one -- `np.isclose` uses a default relative tolerance of (IIRC) 1e-6, so that this ends up being a large number here. Best to set `rtol=0, atol=1e-3` so we testt the times are matching to a msec.